debian/control

```diff
@@ -121,13 +121,48 @@ Description: Translation files for nymead and plugins - translations
  This package provides the translation files for nymead and for the plugins.
 
 
+Package: libnymea-testlib
```

Comment:
libnymea-tests perhaps, then it would be in line with the nymea-tests package below